finagle/buoyant/src/main/scala/com/twitter/finagle/buoyant/TlsServerConfig.scala

```diff
@@ -0,0 +1,45 @@
+package com.twitter.finagle.buoyant
+
+import com.twitter.finagle.Stack
+import com.twitter.finagle.ssl.{ClientAuth => FClientAuth, _}
+import com.twitter.finagle.ssl.server.{SslServerConfiguration, SslServerEngineFactory}
+import com.twitter.finagle.transport.Transport
+import java.io.File
+
+case class TlsServerConfig(
+  certPath: String,
+  keyPath: String,
+  caCertPath: Option[String] = None,
+  ciphers: Option[Seq[String]] = None,
+  requireClientAuth: Option[Boolean] = None
+) {
+  def params(
+    aplnProtocols: Option[Seq[String]],
```

Comment:
whoops, I think this should have been ALPN all along: https://en.wikipedia.org/wiki/Application-Layer_Protocol_Negotiation